src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -168,4 +169,28 @@ public static OwedAmount parseOwedAmount(String owedAmount) throws ParseExceptio
         }
         return new OwedAmount(trimmedOwedAmount);
     }
+
+    /**
+     * Parses a {@code String hoursPaid} into a {@code double}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @throws ParseException if the {@code hoursPaid} is invalid.
+     */
+    public static double parseHoursPaid(String hoursPaid) throws ParseException {
```

Comment:
I think the method name can be better if it is `parseHour`, so can be used for both methods HAHA. And no "s" for being uniform with the `MESSAGE_INVALID_HOUR`

Reply:
Thanks for the suggestion! Ive changed the method name and removed the "s".